### PR TITLE
Extend Info with description in OpenApi model

### PR DESCRIPTION
**Pull request by bmeesters**

### CODE REVIEW

File: openapi/openapi/src/main/scala/endpoints/openapi/model/OpenApi.scala [hunk 1]
```diff
@@ -313,39 +322,45 @@ object OpenApi {
...
     other match {
-      case that: Info => title == that.title && version == that.version
-      case _          => false
+      case that: Info =>
+        title == that.title && version == that.version && description == that.description
+      case _ => false
     }
 
   override def hashCode(): Int =
     Hashing.hash(title, version)
```
Comment (julienrf) [inline openapi/openapi/src/main/scala/endpoints/openapi/model/OpenApi.scala:340]: You should add `description` here as well.

Comment (bmeesters) [reply openapi/openapi/src/main/scala/endpoints/openapi/model/OpenApi.scala:340]: Missed, I'll update.

**Comment by bmeesters** [comment]
Hello @julienrf, I was also wanted to include the tags here, but maybe that is better as a separate PR. WDYT?

**Comment by julienrf** [comment]
Yes, I prefer smaller PRs :) It’s easier to review, thanks!

**Comment by bmeesters** [comment]
Alright, I think this is ready for review. I'll do another PR later for adding support for richer tags.

**Comment by julienrf** [comment]
Thank you!

**Pull request merged**